### ISSUE TITLE
Impossible de choisir des grimoires

### ISSUE THREAD

**Issue by Fouindor** [JulienNigon/SpellcasterUniversity-Bug-Report#216]
v 0.65

![20190216175526_1](https://user-images.githubusercontent.com/15092739/52902830-ac20a280-3216-11e9-863d-bafcf4edf1b6.jpg)

Essayé plusieurs fois à plusieurs endroits différents, impossible de faire quoi que ce soit, je suis obligé de redémarrer le jeu. Le mode sans grimoire marche.

EDIT: ça a marché du premier coup sur la seconde partie.

**Comment by JulienNigon** [comment]
Ouch bizarre. je vais enquêter.

**Comment by nilsbrummond** [comment]
Hello,  I just downloaded and tried the Kickstarter demo.  I get stuck on this blank book selection screen every time I try (same screen as the initial reporter).  I have tried different option but get stuck at this screen every time.  One time I got the screen in non-english ( assume french? not but I can not say for sure ) but still no books to select.

Version 0.68

Installed from the .rar file.
7-Zip info reports: 
Size: 1,811,459,780
Packed Size: 581,626,214
Files: 202
CRC: 2AEC1A45
